distribution/src/bin/elasticsearch

```diff
@@ -24,6 +24,32 @@ ES_JVM_OPTIONS="$ES_PATH_CONF"/jvm.options
 JVM_OPTIONS=`"$JAVA" -cp "$ES_CLASSPATH" org.elasticsearch.tools.launchers.JvmOptionsParser "$ES_JVM_OPTIONS"`
 ES_JAVA_OPTS="${JVM_OPTIONS//\$\{ES_TMPDIR\}/$ES_TMPDIR}"
 
+unset KEYSTORE_PASSWORD
+KEYSTORE_PASSWORD=
+# If keystore exists and is encrypted, prompt for a password
+if [ -f "$ES_PATH_CONF"/elasticsearch.keystore ] \
+    && ! echo $* | grep -E '(^-h |-h$| -h |--help$|--help )' >/dev/null 2>&1 \
+    && ! echo $* | grep -E '(^-V |-V$| -V |--version$|--version )' >/dev/null 2>&1 \
+    && ! "`dirname "$0"`"/elasticsearch-keystore list <&- >/dev/null 2>&1
```

Comment:
This check is true when the keystore is encrypted, but it's also true when `bin/elasticsearch-keystore` isn't where we expect it to be. This seems to be what happened with integration tests. The right thing to do is probably to check that `elasticsearch-keystore` exists before calling it. Then I can remove the new testclusters code.

- [ ] add a check for the `elasticsearch-keystore` script